content/en/platform/corda/4.9/enterprise/node/setup/corda-configuration-fields.md

```diff
@@ -118,7 +118,7 @@ Optional path to the configuration file for the CryptoService provider. This may
 
 Optional time-out value of actions sent to the CryptoService (HSM). If the HSM takes longer than this duration to respond, then a `TimedCryptoServiceException` will be thrown and handled by the Flow Hospital. You can increase it to mitigate the time-out error.
 
-*Default:* 1,000 milliseconds
+*Default:* 10000 milliseconds
```

Comment:
```suggestion
*Default:* 10,000 milliseconds
```

Reply:
I'm not sure if commas work in values so will leave it out in case users get the impression they can be used.